pkgs/development/libraries/graphene/0001-meson-add-options-for-tests-installation-dirs.patch

```diff
@@ -71,7 +71,12 @@ index 77281f5..c4c7fac 100644
        ),
        env: ['MUTEST_OUTPUT=tap'],
        protocol: 'tap',
-@@ -70,13 +79,13 @@ if build_gir and host_system == 'linux' and not meson.is_cross_build()
+@@ -66,17 +75,18 @@ endif
+ if build_gir and host_system == 'linux' and not meson.is_cross_build()
+   foreach unit: ['introspection.py']
+     wrapper = '@0@.test'.format(unit)
++	install_data(unit, install_dir: test_bindir)
```

Comment:
Could you please also open a PR upstream?

Reply:
On it :)

Reply:
https://github.com/NixOS/nixpkgs/pull/125959